package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "root",
   "private": true,
+  "scripts": {
+    "prepare": "git config core.hooksPath .githooks"
```

Comment:
Question: why on the `prepare` script? This is often used when packing a package, or when installing a dependency that needs to be compiled. Should we do this on a different script?